e2e/playwright/tests/todo_plugin.spec.ts

```diff
@@ -6,35 +6,114 @@
 // - [*] indicates an assertion (e.g. * Check the title)
 // ***************************************************************
 
-import {expect, test} from '@e2e-support/test_fixture';
-import SlashCommandSuggestions from 'support/components/slash_commands';
-import {fillMessage, getTodoBotDMPageURL} from 'support/utils';
+import {expect, test} from "@e2e-support/test_fixture";
+import SlashCommandSuggestions from "support/components/slash_commands";
+import {
+    fillMessage,
+    getBotDMPageURL,
+    getLastPost,
+    getTeamName,
+    postMessage,
+} from "support/utils";
+
+const botUserName = "todo";
+let teamName = "";
+
+test.beforeAll(async ({ pw }) => {
+  const { adminClient, adminUser } = await pw.getAdminClient();
+  if (adminUser === null) {
```

Comment:
There's inconsistent indentation in this file. Another thing the linter should be noticing. I'm thinking we should converge on 4 spaces for indenting